extension/src/main/java/com/mobven/extension/Constants.kt

```diff
@@ -0,0 +1,6 @@
+package com.mobven.extension
+
+object Constants {
```

Comment:
Objeye sarmadan da ulaşılabiliyor ama hangisi daha iyi bilmiyorum :D

Reply:
ortak kararla oldugu gibi biraktik. ref: https://stackoverflow.com/questions/44038721/constants-in-kotlin-whats-a-recommended-way-to-create-them